android/java/org/chromium/chrome/browser/crypto_wallet/activities/AssetDetailActivity.java

```diff
@@ -115,10 +115,17 @@ protected void triggerLayoutInflation() {
 
         TextView assetTitleText = findViewById(R.id.asset_title_text);
         assetTitleText.setText(mAssetName);
-        String tokensPath = ERCTokenRegistryFactory.getInstance().getTokensIconsLocation();
-        String iconPath = mAssetLogo.isEmpty() ? null : ("file://" + tokensPath + "/" + mAssetLogo);
-        Utils.setBitmapResource(mExecutor, mHandler, this, iconPath, R.drawable.ic_eth_24, null,
-                assetTitleText, false);
+        if (!mAssetLogo.isEmpty()) {
```

Comment:
similar code block is three times in the PR, here and also in `BuySendSwapActivity.java`.

Maybe it can be moved out into something like `Utils.ChooseAndSetBitmapForAsset()`?

Reply:
let's make it in future refactors of Activity that we plan.

Reply:
:+1: